content/post/go-git-v4.md

```diff
@@ -0,0 +1,73 @@
+---
+author: campoy
+date: 2017-12-07
+title: "Announcing the latest go-git!"
+image: /post/go-git-v4/go-git.png
+description: "After a year of intense work, we're happy to announce the latest and best release of go-git ever. go-git v4 includes many new features, making it the most used and feature complete git library written in Go, and in use on production at companies like source{d} and keybase."
+categories: ["git", "announcement", "OSS", "technical"]
+---
+
+After a bit over a year of intense work, we're happy to announce the
+latest and best release of [go-git](https://github.com/src-d/go-git) yet!
+`go-git v4` includes many new features, making it the most used and feature
+complete git library written in Go.
+
+## what's new?
+
+A year and almost 800 commits have passed since `go-git v3.2.0` was released,
+our amazing contributors have really been very busy. But all of this effort
+was worth it and the results definitely show.
+
+Many (really [many](https://github.com/src-d/go-git/releases/tag/v4.0.0-rc1))
+new features have been added to `go-git`, but our favorites are:
+
+- TODO
+
+In addition to all of these new features *many* bugs were fixed and tests added.
+
+### moving to Apache v2
+
+We always listen to the OSS community, and many voices were asking for a new
+license. As of today we're moving `go-git` to
+[Apache License, Version 2](https://www.apache.org/licenses/LICENSE-2.0).
+
+We hope this will help those wondering whether `go-git` was the good solution
+to their problems!
+
+### we have a logo, and it's adorable!
+
+Following the tradition of many other Go projects, we've adopted the gopher as
+our mascot, and integrate it with the git logo into this amazing creation!
+
+![go-git logo](/post/go-git-v4/go-git-banner.png)
+
+Big shout-out to our designer [Ricardo Baeta](https://github.com/ricardobaeta)
```

Comment:
It seems that the `artwork` repository is being made public these days; if it is done public before publishing this post, the [go-git artwork](https://github.com/src-d/artwork/tree/master/go-git) could be linked from here

Reply:
Yes, it is public now.